src/Symfony/Component/Finder/Iterator/SortableIterator.php

```diff
@@ -41,15 +43,15 @@ public function __construct(\Traversable $iterator, $sort, bool $reverseOrder =
         $order = $reverseOrder ? -1 : 1;
 
         if (self::SORT_BY_NAME === $sort) {
-            $this->sort = function ($a, $b) use ($order) {
+            $this->sort = function (SplFileInfo $a, SplFileInfo $b) use ($order) {
```

Comment:
for class from the global namespace we do not use `use` statements, but add the leading backslash instead

Reply:
this change also looks like it should be applied on the `3.4` branch too

Reply:
or be reverted: it provides no benefit as everything is internal :)

Reply:
It would provide benefits about having code completion by an IDE and finding possible bugs through a static code analysis :-)